GitCommands/DiffMergeTools/Araxis.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitCommands.DiffMergeTools
+{
+    internal class Araxis : DiffMergeTool
+    {
+        /// <inheritdoc />
+        public override string ExeFileName => "Compare.exe";
+
+        /// <inheritdoc />
+        public override string MergeCommand => "/merge /wait /a2 /3 \"$LOCAL\" \"$BASE\" \"$REMOTE\" \"$MERGED\"";
```

Comment:
@hoborm Git uses "$BASE" "$LOCAL" "$REMOTE" "$MERGED"
as default order, even if the support hints say otherwise.
https://www.araxis.com/merge/documentation-windows/integrating-with-other-applications.en#GitExtensions
(seem to be incosistent though)

GE should use the same order as command line, should this be changed?

Reply:
I can definitely say that I have followed the araxis help page resolving this one, should I submit an other PR with the corrected order? Also I would suggest to contact them to correct their documentation as well since it is misleading in this case.